sdcm/cluster.py

```diff
@@ -542,6 +542,23 @@ def is_server_encrypt(self):
         result = self.remoter.run("grep '^server_encryption_options:' /etc/scylla/scylla.yaml", ignore_status=True)
         return 'server_encryption_options' in result.stdout.lower()
 
+    def extract_seeds_from_scylla_yaml(self):
+        yaml_dst_path = os.path.join(tempfile.mkdtemp(prefix='sct'), 'scylla.yaml')
+        self.remoter.receive_files(src=SCYLLA_YAML_PATH, dst=yaml_dst_path)
+        node_seed_ips = []
+        with open(yaml_dst_path, 'r') as yaml_stream:
+            conf_dict = yaml.safe_load(yaml_stream)
```

Comment:
the logic in this blcok is incorrect:
1. First, you need to try to parse the YAML file (wrap with `try..except` the `yaml.safe_load`)
2. Handle getting the seeds

Print error messages in the right `try..except` block

Reply:
fixed